pkg/meshsync/service/service.go

```diff
@@ -1 +1,49 @@
 package service
+
+import (
+	"log"
+
+	discovery "github.com/layer5io/meshery-operator/pkg/discovery"
+	clusterPipelinePackage "github.com/layer5io/meshery-operator/pkg/meshsync/cluster/pipeline"
+	istioPipelinePackage "github.com/layer5io/meshery-operator/pkg/meshsync/meshes/istio/pipeline"
+	"k8s.io/client-go/rest"
+)
+
+// StartDiscovery - run pipelines
+func StartDiscovery() {
```

Comment:
@kumarabd , Should this function return any error?

Reply:
yes ideally because this is not going to be the `main` function